authorizations.md

```diff
@@ -54,6 +54,23 @@ Return codes:
 * 401 Unauthorized - if you are not authenticated and an eperson is specified
 * 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators and the user specified in the eperson parameter can access
 
+#### objects
+**/api/authz/authorizations/search/objects?uri=<:uri>[&eperson=<:uuid>&feature=<:string>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uri: mandatory, epeatable. Represents the list of objects to use for the authorization check. For each of them, the full URI of the rest resource must be specified, i.e. https://{dspace.url}/api/core/community/{uuid}
```

Comment:
Per today's meeting discussion, I just wanted to document we decided to move to `uuid` in the URL instead of the full `uri`.  

@benbosman noted that per his research several web servers *still have URL size limitations*.  They are usually large (in 4,000 or so range) and usually configurable, but we'd like to avoid anyone encountering those limitations with the DSpace REST API.